### ISSUE TITLE
Flatten-Tool has no output if MemoryError

### ISSUE THREAD

**Issue by jpmckinney** [OpenDataServices/flatten-tool#321]
See CRM issue 5026. The `-v` option had to be used to turn on this output. It is not a good user experience to have a command run for a long time without any positive or negative side effects.

I think it'd be a good idea to have `-v` on by default, and to instead require users to opt for quiet output.

**Comment by robredpath** [comment]
>  It is not a good user experience to have a command run for a long time without any positive or negative side effects. 

Agreed, although I'm also wary of the potential for disorientation if a user sees a load of warnings from flatten-tool internals, but the operation that they have tried to carry out has basically worked, and so the warnings can be ignored. For an experienced user, that's no problem - but some of flatten-tool's users are completely new to the command line. 

I feel like this is something that we've not spent much time on refining, so I suspect that there are 'easy wins' to be had in quickly cleaning up the commandline interface both in general, and specific for OCDS, as we've discussed recently. 

To that end, I've added a Trello card (see below) to select a few things like this to look into, and if we decide to star it when we next talk, start to develop
